hazelcast/src/test/java/com/hazelcast/internal/management/ClearWanQueuesTest.java

```diff
@@ -0,0 +1,62 @@
+package com.hazelcast.internal.management;
+
+import com.eclipsesource.json.JsonObject;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.instance.Node;
+import com.hazelcast.internal.management.operation.ClearWanQueuesOperation;
+import com.hazelcast.internal.management.request.ChangeWanStateRequest;
+import com.hazelcast.internal.management.request.ClearWanQueuesRequest;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class ClearWanQueuesTest extends HazelcastTestSupport {
+
+    private ManagementCenterService managementCenterService;
+
+    @Before
+    public void setUp() {
+        HazelcastInstance hz = createHazelcastInstance();
+        Node node = getNode(hz);
+        managementCenterService = node.getManagementCenterService();
+    }
+
+    @Test
+    public void testOperationDefaultConstructor() {
+        ClearWanQueuesOperation operation = new ClearWanQueuesOperation();
```

Comment:
Testing a default constructor isn't providing a lot of value since it is designed to do nothing.

Also no verification is done on the state (which would be pointless anyway). But verifying of the created object really is created, is not proving much value since otherwise we would have a major JVM bug at our hands.

I would remove this test since it doesn't provide any value. And in the testSerialization you are already testing the default constructor due to the serialization.